### DISCUSSION TITLE
ICIP 2021 paper on Surface Defect Recognition

### DISCUSSION THREAD

**Discussion by vikxoxo** [Charmve/Surface-Defect-Detection#10] [Show and tell]
Dear Connections, 
I hope you are doing well.

I am happy to share that our #research work has been accepted in 2021 IEEE  International Conference on Image Processing (ICIP), 19-22 September 2021 • Anchorage, Alaska, USA
and available at below link as an open preview.

https://lnkd.in/gW-t_qEg

Or

https://ieeexplore.ieee.org/document/9506405

● In this work we have proposed S2D2Net, an automatic defect recognition network for surface defect diagnosis, achieving SOTA results on publicly available steel surface defect datasets (NEU-CLS and extended diversified dataset ENEU).

Our method utilizes CapsNet over extracted feature maps from a pretrained ImageNet model, enabling faster training, robustness, and learning with very few samples.

● We welcome researchers working in different vision domains to analyse the scope of our hybrid technique in thier own applications targeting small sample problems along with robustness.

I invite readers to please review, and cite our work if found useful.
Thanks.

P.S. ICIP is a tier-A core CSE Conference having h5-index>60 on Google Scholar metrics, and features among the top publication under "Computer Vision and Pattern Recognition" category.

Links: 

Google Scholar:
https://scholar.google.co.in/citations?view_op=top_venues&hl=en&vq=eng_computervisionpatternrecognition

Top Conferences in Core CSE:
http://cic.tju.edu.cn/faculty/zhileiliu/doc/COREComputerScienceConferenceRankings.html

ICIP 2021 official site: 
https://www.2021.ieeeicip.org/

 #artificialintelligence #computervision #ai #industria40 #deeplearning #ieeexplore #vision #visioninspection #AI #computervision #deeplearning #TensorFlow #surfacepattern #patternrecognition

**Comment by Charmve** [comment]
Hi. Vikanksh Nath

wellcome all your fantastic work on this forum.

but this url does not work, please check it.

**Comment by vikxoxo** [comment]
May be it's due to location.

Kindly see the below link:

https://ieeexplore.ieee.org/document/9506405

**Reply by Charmve** [reply]
this url is ok, I can visit it in my city.

more importantly, could you please share the souce code and dataset with us. 

if so, your work should be collected in this repo.

Thanks Vikanksh Nath.
